dockerfiles/Dockerfile

```
@@ -0,0 +1,84 @@
+FROM ubuntu:latest
+LABEL authors="julius"
+
+# get build-arg inputs to set the singularity provider and versions
+ARG SINGULARITY_PROVIDER
+ENV SINGULARITY_PROVIDER=${SINGULARITY_PROVIDER}
+ARG SINGULARITY_VERSION
+ENV SINGULARITY_VERSION=${SINGULARITY_VERSION}
+
+ENV DEBIAN_FRONTEND=noninteractive
```

Maybe set this as an ARG instead of ENV to make it only set during build time as discussed here: https://serverfault.com/a/797318
```suggestion
ARG DEBIAN_FRONTEND=noninteractive
```